docs/source/usage/python.rst

```diff
@@ -807,7 +807,7 @@ This module provides elements for the accelerator lattice.
    :param dy: vertical translation error in m
    :param rotation: rotation error in the transverse plane [degrees]
 
-.. py:class:: impactx.elements.ShortRF(V, freq, phase, dx=0, dy=0, rotation=0)
+.. py:class:: impactx.elements.ShortRF(V, freq, phase=-90.0, dx=0, dy=0, rotation=0)
```

Comment:
We have no default parameter for the phase.
```suggestion
.. py:class:: impactx.elements.ShortRF(V, freq, phase, dx=0, dy=0, rotation=0)
```

Reply:
Oh, we have in the inputs file but not in Python. Can you please add the default of `-90.0` in `src/python/elements.cpp`?

Reply:
I was wrong, it's added there already.